### ISSUE TITLE
Animation on blocks with CTM not working

### ISSUE THREAD

**Issue by second2050** [PepperCode1/Continuity#33]
While playing with [BetterVanillaBuilding](https://www.curseforge.com/minecraft/texture-packs/bettervanillabuilding/) I realized that especially fire and campfire blocks are not animated anymore and are stuck on a seemingly random frame of their animation.

I can work around it by deleting the files in the resource pack that hold the CTM information for the blocks in `/assets/minecraft/optifine/ctm`

I only encountered that bug with Continuity installed, disabling CTM in Continuity's settings did not fix it.

My Setup is:
- Minecraft 1.18
- Continuity 1.0.3
- Sodium + Indium + Iris
- ![image](https://user-images.githubusercontent.com/37344371/144940249-37a45ddb-2863-44a2-a359-345acefb1e5a.png)

**Comment by PepperCode1** [comment]
I'm pretty sure this is an Indium bug. Test with just Fabric API and Continuity.

**Comment by second2050** [comment]
Okay I tested it with just Continuity and Fabric API and it works, so it is indeed a bug with Indium.

For other people looking for a solution:
I checked the indium issue tracker and found [issue #83](https://github.com/comp500/Indium/issues/83) which describes this;
tl;dr: to get this to work I need to disable "Animate Only Visible Textures" in Sodium's settings.
